packages/calcite-components/src/components/action-group/action-group.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 .container {
-  @apply flex flex-col;
+  @apply flex flex-col flex-grow;
```

Comment:
Sidebar: WDYT about doing an audit and possibly defining guidelines on expected styles on the root internal container? cc @macandcheese @geospatialem